andela/ah-groot-frontend
# 165019446 Deploying to heroku

**What does this PR do?**
* Deploys application to heroku

**Description of the tasks to be completed?**
* Add the static.json file
* Add the commands for running the deployment
* Add the required engines that run node version
* Add the npm dependency

**How should this be manaully tested?**
* Fetch the branch using git fetch origin `ch-heroku-deploy-165019446`
* Check into the branch using git checkout `ch-heroku-deploy-165019446`
* Run npm install npm && npm install to install the dependencies
* Run `npm run-script build` to start the development server
* Navigate to https://ah-front-end-groot.herokuapp.com/

**What are the relevant pivot tracker stores?**
#165019446
<img width="1440" alt="Screenshot 2019-04-02 at 13 11 38" src="https://user-images.githubusercontent.com/42295880/55398307-e0df8380-5550-11e9-9d15-b1322d1f2f6d.png">
